app/controllers/catalog_controller.rb

```diff
@@ -1968,7 +2002,7 @@ def validate_vm_name
   end
 
   def automate_tree_needed?
-    params[:display] || params[:template_id] || params[:manager_id] || params[:ovf_template_id]
+    params[:display] || params[:template_id] || params[:manager_id] || params[:ovf_template_id] || params[:datacenter_id] || params[:resource_pool_id] || params[:ems_folder_id] || params[:host_id] || params[:storage_id]
```

Comment:
@DavidResende0 Is it possible to ass all these variables names in an array and loop through?